package.nls.json

```diff
@@ -53,6 +53,9 @@
     "AWS.command.quickStart": "View Quick Start",
     "AWS.command.quickStart.title": "AWS Toolkit - Quick Start",
     "AWS.command.quickStart.error": "There was an error retrieving the Quick Start page",
+    "AWS.credentials.statusbar.no.credentials": "(not connected)",
+    "AWS.credentials.statusbar.text": "AWS Credentials: {0}",
```

Comment:
Not sure if I'm a fan of this messaging. I think I'd prefer something completely different from how a set credential appears; make the whole displayed message read "Not connected to AWS" or similar.

The absolute worst case scenario is that a user has a credential profile named `(not connected)`

Reply:
> make the whole displayed message read "Not connected to AWS" or similar.

My only concern here is to give it text that is still meaningful in the context of looking at VS Code as a whole. By keeping the same format, the aim is that it resonates more that it relates to the AWS Toolkit. Perhaps the tooltip helps with that though.

> The absolute worst case scenario is that a user has a credential profile named (not connected)

I put very low odds on this happening.

Reply:
Maybe "No AWS Credentials"? Still gets across that this is an area for AWS credentials but puts the fact that there aren't set credentials first. Not a hill I'm willing to die on, though; it just feels like slightly less mental energy expended.

Reply:
This is what the JetBrains toolkit shows:

![image](https://user-images.githubusercontent.com/39839589/71373304-992fda80-256b-11ea-8f85-e4cca9058f15.png)

How do you feel about `AWS Credentials: None selected` (retaining the `AWS Credentials: {0}` format)

Reply:
^ That someone is going to name their credentials "None selected" to mess with someone :P 

Again, not a hill I'm willing to die on, I think I like the parenthesis more than this if you're dead set on keeping the format.